### DISCUSSION TITLE
Telegram Notifications issue

### DISCUSSION THREAD

**Discussion by galulex** [zepp-health/zeppos-forum#325] [Developer Feedback]
Telegram duplicates first notification

There are two notifications on my phone
On the Amazfit Balance there are three notifications
First one is duplicated

Apps like Slack works fine

![photo1711619532](https://github.com/zepp-health/zeppos-forum/assets/483482/7867f18e-c304-4b45-bf1d-c8668a294c4f)
![photo1711619532 (3)](https://github.com/zepp-health/zeppos-forum/assets/483482/74724ed6-17e8-4d77-83a5-824aa8dd0ae2)
![photo1711619532 (2)](https://github.com/zepp-health/zeppos-forum/assets/483482/54cb8506-cbea-4071-a3d2-6ba6863e0bc9)
![photo1711619532 (1)](https://github.com/zepp-health/zeppos-forum/assets/483482/fc6c3d88-dd2c-40bb-bd80-e37df7c271aa)

**Comment by MyoungXUE** [comment]
Hi. This looks like a bug. Relevant colleagues will investigate it. Thank you for your feedback!

**Comment by MyoungXUE** [comment]
Please refer to this document to initiate a user feedback.

https://docs.zepp.com/docs/guides/faq/upload-log/

**Comment by albertincx** [comment]
It seems that for the Google Play version there is no such issue :champagne:  (tested Tg X and GPlay version)

**Reply by albertincx** [reply]
Now tested on apk version. It works. @galulex can you provide steps to repeat?

**Reply by galulex** [reply]
it is not fixed. Use just telegram not tg x
Receive two messages in different chats. You will see three notifications on the watch
